frontend/src/PagesComponents/Organization_/Project_/Updates/Table.elm

```diff
@@ -130,10 +132,14 @@ showAllTables now from erd =
     )
 
 
-hideTable : Time.Posix -> TableId -> Erd -> Erd
+hideTable : Time.Posix -> TableId -> Erd -> ( Erd, Maybe ( Msg, Msg ) )
 hideTable now id erd =
+    -- FIXME: must keep table details in history to get back color, shown columns and so on
```

Comment:
Looks like I can't reuse existing `Msg` as I miss data, will need specific ones for undo :/